queries.sql

```diff
@@ -140,20 +145,182 @@ vet-clinic=# SELECT * FROM animals WHERE neutered = true;
 
 --Find all animals not named Gabumon.
 
-vet-clinic=# SELECT * FROM animals WHERE NOT name = 'Gabumon';
+SELECT * FROM animals WHERE NOT name = 'Gabumon';
 -- id |  name   | date_of_birth | escape_attempts | neutered | weight_kg 
 ----+---------+---------------+-----------------+----------+-----------
 --  2 | Agumon  | 2020-02-03    |               0 | t        |     10.23
 --  4 | Pikachu | 2021-01-07    |               1 | f        |     15.04
 --  5 | Devimon | 2017-05-12    |               5 | t        |        11
 --(3 rows)
 
-
 --Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights --that equals precisely 10.4kg or 17.3kg)
-vet-clinic=# SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
 -- id |  name   | date_of_birth | escape_attempts | neutered | weight_kg 
 ----+---------+---------------+-----------------+----------+-----------
 --  4 | Pikachu | 2021-01-07    |               1 | f        |     15.04
 --  5 | Devimon | 2017-05-12    |               5 | t        |        11
 --(2 rows)
 
+
+-- PLEASE NOTE THAT PROJECT REQUIRMENTS START FROM HERE
+
+--Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that species columns went back to the state before tranasction.
+BEGIN;
+BEGIN
+UPDATE animals SET species = 'unspecified';
+UPDATE 10
+ROLLBACK;
+ROLLBACK
+
+vet-clinic=# SELECT * FROM animals;
+ id |    name    | date_of_birth | escape_attempts | neutered | weight_kg | species 
+----+------------+---------------+-----------------+----------+-----------+---------
+  2 | Agumon     | 2020-02-03    |               0 | t        |     10.23 | 
+  3 | Gabumon    | 2018-11-15    |               2 | t        |         8 | 
+  4 | Pikachu    | 2021-01-07    |               1 | f        |     15.04 | 
+  5 | Devimon    | 2017-05-12    |               5 | t        |        11 | 
+  6 | Charmander | 2020-02-08    |               0 | f        |       -11 | 
+  7 | Plantmon   | 2022-11-15    |               2 | t        |      -5.7 | 
+  8 | Squirtle   | 1993-04-02    |               3 | t        |    -12.13 | 
+  9 | Angemon    | 2005-06-12    |               1 | t        |       -45 | 
+ 10 | Boarmon    | 2005-06-07    |               7 | t        |      20.4 | 
+ 11 | Blossom    | 1998-10-13    |               3 | t        |        17 | 
+(10 rows)
+
```

Comment:
- [ ] It would also be great that you don't present the result of your query outcome as code in your main project, instead if you must it is better it is provided as JPEG in your documentation. Therefore I will advice please you kindly take it out also since we can't comment on and them, what do you think?